src/imagery/i.eodag/i.eodag.html

```diff
@@ -2,38 +2,272 @@ <h2>DESCRIPTION</h2>
 <h3>WARNING: I.EODAG IS UNDER DEVELOPMENT. THIS IS AN EXPERIMENTAL VERSION.</h3>
 
 <em>i.eodag</em> allows to search and download imagery scenes, e.g. Sentinel,
-Landsat, and MODIS, from a number of different providers.
-The module utilizes the
+Landsat, and MODIS, as well as other Earth Observation products, from a number
+of different providers. The module utilizes the
 <a href="https://eodag.readthedocs.io/en/stable/">EODAG API</a>,
-as a single interface to search for scenes on the providers that EODAG supports.
-
+as a single interface to search for products within the supported providers.
 
 <p>
-Currently, only products which footprint intersects the current computational
-region extent (area of interest, AOI) are retrieved.
+By default <em>i.eodag</em> will search for products which footprint intersects
+the current computational region extent. Users can alternatively opt to pass a
+vector map throught the <b>map</b> option to define the area of interest (AOI)
+and change the relation with product footprints by means of the
+<b>area_relation</b> or <b>minimum_overlap</b> options.
 
 <p>
 To only list available scenes, <b>l</b> flag must be set. If no <b>start</b>
 or <b>end</b> dates are provided, the module will search scenes from the
 past 60 days.
 
 <p>
-To download all scenes found within the time frame provided, the user must
+To download all scenes found within the time frame provided, users must
 remove the <b>l</b> flag and provide an <b>output</b> directory. Otherwise,
-files will be downloaded into <em>/tmp</em> directory.
+files will be downloaded into the /tmp directory.
 To download only selected scenes, one or more IDs must be provided
 through the <b>id</b> option.
 
 <p>
-To be able to download data through <em>i.eodag</em>, the user will need to register for the
-providers of interest.
-<em>i.eodag</em> reads the user credentials, which the user should have acquired,
-from the EODAG YAML config file.  User have to specify the config file path
-location through the <b>config</b> option, otherwise <em>i.eodag</em>
-will use the credentials found in the default config file <em>~/.config/eodag/eodag.yml</em>
+To be able to download data through <em>i.eodag</em>, users will
+need to register for the providers of interest.
+<em>i.eodag</em> reads user credentials from the EODAG YAML config file.
+Users have to specify the config file path through the <b>config</b>
+option, otherwise <em>i.eodag</em> will use the credentials found in
+the default config file <em>~/.config/eodag/eodag.yml</em> which is auto-generated the
+first time eodag is used after an install in `$HOME/.config/eodag/eodag.yml`.
```

Comment:
```suggestion
Users have to specify the configuration file path through the <b>config</b>
option, otherwise <em>i.eodag</em> will use the credentials found in
the default config file which is auto-generated the first time EODAG is used 
after installation. The configuration file is stored by default in 
<code>$HOME/.config/eodag/eodag.yml</code>.
```